apps/backend/src/common/services/user_service.py

```
@@ -0,0 +1,509 @@
+            # Update password
+            user.password_hash = self.password_service.hash_password(new_password)
+            user.password_changed_at = datetime.utcnow()
+            
+            # Mark token as used
+            verification.mark_as_used()
+            
+            # Invalidate all user sessions
+            sessions_result = await db.execute(
+                select(Session).where(
+                    Session.user_id == user.id,
+                    Session.is_active == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if Session.is_active:` for truth checks

Replace with `Session.is_active`

(E712)